docs/operations/node_assignment.md

```diff
@@ -140,3 +140,17 @@ An example for `tolerations` may look like this:
         operator: "Equal"
         value: "value"
         effect: "NoSchedule"
+
+## Node balancing with Descheduler
+
+In some cases we might need to rebalance the cluster on current scheduling policy 
+and load conditions. [Descheduler](https://github.com/kubernetes-sigs/descheduler) 
+can find pods, which violates e.g. scheduling decisions and evict them based on descheduler 
+policies. Kubevirt VMs are handled as pods with local storage, so by default, 
+descheduler will not evict them. But it can be easily overridden by adding special annotation to vm:
+
+    descheduler.alpha.kubernetes.io/evict: true
```

Comment:
This annotation would not be passed on to the pod (well maybe in CNV 1.4 where there was a bug around that, but not no for a very long time).